test/integration/tcp_tunneling_integration_test.cc

```diff
@@ -176,8 +192,7 @@ TEST_P(ConnectTerminationIntegrationTest, UsingHostMatch) {
   exact_match_ = true;
   initialize();
 
-  connect_headers_.removePath();
-  connect_headers_.removeProtocol();
+  clearExtendedConnectHeaders();
```

Comment:
looks like we only use this once - mind doing changes inline?
isn't this really just using connect headers instead of extended connect?

Reply:
Done